src/core/etl/src/Flow/ETL/DSL/Entry.php

```diff
@@ -276,27 +262,32 @@ final public static function string_upper(string $name, string $value) : RowEntr
         return RowEntry\StringEntry::uppercase($name, $value);
     }
 
-    final public static function struct(string $name, RowEntry ...$entries) : RowEntry
-    {
-        return self::structure($name, ...$entries);
-    }
-
     /**
      * @throws InvalidArgumentException
      *
      * @return RowEntry\StructureEntry
      */
     final public static function structure(string $name, RowEntry ...$entries) : RowEntry
```

Comment:
This approach is no longer valid since the structure was refactored. What it should take instead StructType and array of values